wiki/Modding/MAT_Leaders/en.md

```diff
@@ -0,0 +1,19 @@
+---
+tags:
+  - Mapping Assistance Team Leaders
+---
+
+# MAT Leaders
+
+**MAT Leaders** oversaw the [Mapping Assistance Team](/wiki/Modding/Mapping_Assistance_Team) (*MAT*). They were responsible for managing and mentoring team members, as well as raising awareness of important announcements and shaping the future goals and direction of the team. There was only one MAT leader at a time.
+
+## History
+
+![][flag_AU] [Ephemeral](https://osu.ppy.sh/users/102335) was chosen as the inaugural MAT Leader at the group's formation on the 29th of December, 2009.
```

Comment:
either this is the incorrect date, or it should say "shortly after the group's formation", because the group was formed on the 27th

Reply:
Correct, this should be the 27th